CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [3.6.2 - Xcode 11.4 on ???, 2020](https://github.com/CocoaLumberjack/CocoaLumberjack/releases/tag/3.6.2)
 
 ### Public
+- **Breaking change**: Dropped support for iOS 8 (#1153)
```

Comment:
As mentioned: depending on when we release what version, this might need to go into another section... This is merely a reminder note, though. No action needed at the moment 🙂